tools/db_crashtest.py

```diff
@@ -846,14 +846,8 @@ def exit_if_stderr_has_errors(stderr, print_stderr=True):
 
     stderrdata = stderr.lower()
     errorcount = stderrdata.count("error") - stderrdata.count("got errors 0 times")
-    print("#times error occurred in output is " + str(errorcount) + "\n")
+    print("For debugging - #times std::error occurred in output is " + str(errorcount) + "\n")
```

Comment:
The changes to this `print()` are confusing. Can we either delete it or revert it?